esphome/components/mcp4461/output/mcp4461_output.cpp

```diff
@@ -12,6 +12,13 @@ static const char *const TAG = "mcp4461.output";
 // floats from other components (like light etc.) are passed as "percentage floats"
 // this function converts them to the 0 - 256 range used by the MCP4461
 void Mcp4461Wiper::write_state(float state) {
+  if (!std::isfinite(state)) {
```

Comment:
```suggestion
  if (!std::isfinite(state)) {
  ESP_LOGW.....
```

Could log the warning that it requires finite value. Just a suggestion, not really important.